tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/CiamIntegrationTests.cs

```diff
@@ -144,6 +149,74 @@ public async Task ClientCredentialWithClientSecret_Ciam_Async(string authorityFo
             Assert.AreEqual(TokenSource.Cache, result.AuthenticationResultMetadata.TokenSource);
         }
 
+        [TestMethod]
+        public async Task Ciam_Cud_Obo_Test_Async()
+        {
+            string authorityNonCud = "https://MSIDLABCIAM6.ciamlogin.com";
+            string authorityCud = "https://login.msidlabsciam.com/fe362aec-5d43-45d1-b730-9755e60dc3b9/v2.0/";
+            string ciamWebapp = "b244c86f-ed88-45bf-abda-6b37aa482c79";
+            string ciamWebApi = "634de702-3173-4a71-b336-a4fab786a479";
+            string ciamEmail = "idlab@msidlabciam6.onmicrosoft.com";
```

Comment:
There is a "GetLabUser(username)" API since you are anyway hardcoding the lab user.

Reply:
This is a brand new user and it isnt in the lab api at this time

Reply:
I asked the lab when ths api will be available. if it is not available soon we can revisit.